src/main/java/com/bergerkiller/bukkit/tc/signactions/SignActionType.java

```diff
@@ -1,40 +1,43 @@
 package com.bergerkiller.bukkit.tc.signactions;
 
 public enum SignActionType {
-    NONE, REDSTONE_CHANGE, REDSTONE_ON, REDSTONE_OFF, MEMBER_ENTER, MEMBER_MOVE, MEMBER_LEAVE, GROUP_ENTER, GROUP_LEAVE,
-    MEMBER_UPDATE, GROUP_UPDATE;
+    NONE(false, true), 
+    REDSTONE_CHANGE(true, false), 
+    REDSTONE_ON(true, false), 
+    REDSTONE_OFF(true, false),  
+    MEMBER_ENTER(false, true),  
+    MEMBER_MOVE(false, true),  
+    MEMBER_LEAVE(false, true),  
+    GROUP_ENTER(false, true),  
+    GROUP_LEAVE(false, true), 
+    MEMBER_UPDATE(false, false),  
+    GROUP_UPDATE(false, false);
+    
+    
+    private signActionType(boolean redstone, boolean movement) {
```

Comment:
I would assume `private signActionType(...)` should give an error because of case sensitivity?

Reply:
You are absolutely right! I used the built-in editor from github :( 
I will change it.
Thank you c: